src/mscorlib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -864,6 +864,14 @@ public static void PrelinkAll(Type c)
             }
         }
 
+        internal static int NumParamBytes(RuntimeMethodInfo m, bool isForStdCallDelegate)
```

Comment:
This is based somewhat on the full framework public API (https://msdn.microsoft.com/en-us/library/system.runtime.interopservices.marshal.numparambytes(v=vs.110).aspx).